db-developments/sql/aggregate/spatial.sql

```diff
@@ -0,0 +1,90 @@
+DROP TABLE IF EXISTS aggregate_{{ geom }}_{{ decade }} CASCADE;
```

Comment:
This file replaces the 6 or so spatial aggregate files (tract.sql etc). Called/formatted by aggregate.py